dictionaries/en_GB-ise/README.md

```diff
@@ -0,0 +1,63 @@
+# CSpell British English (-ise) Dictionary
+
+British English dictionary for cspell.
+
+Use this dictionary if you do not want the (-ize) spelling variants commonly used in US English.
+
+## Requirements
+
+| Tool                                                                                                                                 | Version |
+| ------------------------------------------------------------------------------------------------------------------------------------ | ------- |
+| [cspell](https://github.com/streetsidesoftware/cspell)                                                                               | `>= 6`  |
+| [Code Spell Checker - Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) | `>= 2`  |
+
+## Installation
+
+Global Install and add to cspell global settings.
+
+```sh
+npm install -g @cspell/dict-en-gb
+cspell link add @cspell/dict-en-gb
+```
+
+## Uninstall from cspell
+
+```sh
+cspell link remove @cspell/dict-en-gb
+```
```

Comment:
Wrong module name, no?

It should be en_GB-ise

Reply:
fixed in #4357